docs/API/后端API/AI功能接口.md

```
@@ -0,0 +1,49 @@
+
+**Method：** POST
+
+**接口描述：**
+
+<p>异常响应实例</p>
+<p><span class="colour" style="color:rgb(0, 0, 0)">{ "data":</span><span class="colour" style="color:rgb(0, 128, 128)"><span class="colour" style="color:rgb(0, 128, 128)">null</span></span><span class="colour" style="color:rgb(0, 0, 0)">, "locale":</span><span class="colour" style="color:rgb(221, 17, 68)"><span class="colour" style="color:rgb(221, 17, 68)">"zh-cn"</span></span><span class="colour" style="color:rgb(0, 0, 0)">, "error": { "code":</span><span class="colour" style="color:rgb(221, 17, 68)"><span class="colour" style="color:rgb(221, 17, 68)">"CM002"</span></span><span class="colour" style="color:rgb(0, 0, 0)">, "message":</span><span class="colour" style="color:rgb(221, 17, 68)"><span class="colour" style="color:rgb(221, 17, 68)">"参数错误"</span></span><span class="colour" style="color:rgb(0, 0, 0)">}, "err_msg":</span><span class="colour" style="color:rgb(221, 17, 68)"><span class="colour" style="color:rgb(221, 17, 68)">"Additional properties are not allowed ('role1' was unexpected) - 'messages.0'"</span></span><span class="colour" style="color:rgb(0, 0, 0)">}</span></p>
+
+
+### 请求参数
+
+**Headers**
```

Markdownlint (0.35.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)